updated and corrected articles

elizabethcbailey opened the pull request:
removed html comments, removed H1 headings, updated image and copy in SmartGeo article, removed dropshadow from thumbnail images for getting started.

juliaklusek commented on news/smart-geo/details.md:
```diff
@@ -1,14 +1,16 @@
-# Did you catch us at Smart Geotechnics?
 
-![Oasys team at Smart Geotechnics](https://b2c-templates-arup.s3-eu-west-1.amazonaws.com/gofer/Smart_Geo_Image_Gofer.svg)
+![Photo of: Norbert Kovacs - Oasys, Phil Child - Seequent, Mark Skinner - Oasys, in front of Giraphe booth](https://b2c-templates-arup.s3-eu-west-1.amazonaws.com/giraphe/Smartgeo-Seequent-and-Oasys.jpg)
```
The new image isn't displayed correctly because it's too big - it should be under 5MB. Uploading it as SVG may help :)

elizabethcbailey replied:
corrected in second commit.

juliaklusek replied:
The photo seems to have some sort of filter on it. Can you check if you see it too?

juliaklusek reviewed (changes requested):
The changes looks good - only the smart-geo image needs to be updated per earlier comment and the webinar article has an HTML comment.

I've also noticed that some markdown files start with a blank line - is it a markdown requirement or could the image be on line 1?

elizabethcbailey commented:
> The changes looks good - only the smart-geo image needs to be updated per earlier comment and the webinar article has an HTML comment.
> 
> I've also noticed that some markdown files start with a blank line - is it a markdown requirement or could the image be on line 1?
I could not find the html comment left in there but in any case the content in the webinar details is just placeholder text and will be updated with the real article later early next week. 
Image has been converted and updated.
removed blank line above image in second commit.

juliaklusek commented on news/getting-started/details.md:
```diff
@@ -1,30 +1,23 @@
 Giraphe is the latest cloud-based solution from Oasys. Smart, powerful, and user-friendly: welcome to geotechnical graphing for the digital age. Produced in partnership with Seequent, Giraphe takes the hard work out of graphing, communicating seamlessly with OpenGround data and automatically saving your creations. Create, refresh and export report-ready graphs from wherever you have an internet c ...
 
 ## What’s in a name? 
-
```
The standard is to put blank lines before and after headings, so it was good before :) It shouldn't break anything now, but let's fix it on the next PR.

juliaklusek reviewed (changes requested):
The photo issue must be looked into.
